.github/workflows/slack-office-hours-dev.yml

```diff
@@ -68,7 +68,7 @@ jobs:
                       "emoji": true
                     },
                     "value": "click_me_123",
-                    "url": "https://ec.yourlearning.ibm.com/w3/enrollment/event/10322733",
+                    "url": "https://ec.yourlearning.ibm.com/w3/enrollment/event/10408799",
```

Comment:
For the Dev Office Hours link, do you want this to link to "enrollment" or just to the event, like you are doing for Design Office Hours? Just noticed the two links were slightly different in that way.

Reply:
Yeah this just cuts a step out of the process and immediately enrolls them when they click the link

Reply:
ok cool!